src/views/EnrollmentCheckIn/components/EnrollmentCheckInWelcome/components/MajorHolds/index.jsx

```diff
@@ -1,9 +1,8 @@
 import { Grid, Typography } from '@mui/material';
-import { gordonColors } from 'theme';
 
 const MajorHolds = ({ holds }) => (
   <Grid item>
-    <Typography style={{ color: gordonColors.primary.blue }} align="center" variant="h6">
+    <Typography color="primary" align="center" variant="h6">
```

Comment:
Have you tested this? The `Typography` component has no `color` prop, so I don't think this works.

Reply:
Yes, we have tested it, and we've used it in a bunch of places.  It sets the text color just fine @EjPlatzer

Reply:
Super helpful that they don't have this officially documented, lol...